kiwi/defaults.py

```diff
@@ -658,6 +655,17 @@ def get_default_boot_timeout_seconds(self):
         """
         return 10
 
+    @classmethod
+    def get_default_disk_start_sector(self):
+        """
+        Implements the default initial disk sector for the first disk
+        partition.
+
+        :return: sector
+        :rtype: int
+        """
+        return 2048
```

Comment:
Should be:

```python
return self.defaults['kiwi_startsector']
```

Reply:
Well, it should self initate first, which is a bit odd, but it perfectly works, in fact, this was my first attempt.